payments-core/src/main/java/com/stripe/android/networking/StripeRepository.kt

```diff
@@ -548,7 +548,7 @@ abstract class StripeRepository {
     ): SetupIntent?
 
     @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-    abstract suspend fun getPaymentMethodMessaging(
+    abstract suspend fun retrievePaymentMethodMessaging(
```

Comment:
With this returning a `PaymentMethodMessage`, should the name be `retrievePaymentMethodMessage()`?